best_bank_as/models/account.py

```diff
@@ -76,12 +76,12 @@ def get_transactions(self) -> list[dict[str, Any]]:
         return transactions
 
     @classmethod
-    def request_new_account(cls, customer: "Customer") -> "Account":
+    def request_new_account(cls, customer: "Customer", status: int) -> "Account":
```

Comment:
where is `status` being used? we could maybe send an enum instead of an int? makes it more readable